doom.config.yml

```
@@ -49,3 +49,57 @@ releaseNotes:
 
 # onlyIncludeRoutes:
 #   - '*/developer/building_application/functions/namespace/**'
+
+
+exports:
+  - name: overview
+    scope:
+    - '*/overview/'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)